src/main.py

```diff
@@ -1,25 +1,22 @@
 '''main executable for setting up VTOL'''
 import json
-from dronekit import LocationGlobalRelative
-from vtol import setup_vehicle
+from util import setup_vehicle
+from vtol import VTOL
 
 def main(configs):
     '''Configure vtol and ready for mission'''
-    vehicle = setup_vehicle(configs)
-
+    vehicle = setup_vehicle(configs, VTOL)
     vehicle.takeoff()
 
-    vehicle.set_altitude(3)
+    # vehicle.send_ned_velocity(-5, 3, 0, 5)
+    vehicle.set_attitude(pitch_angle=20, duration=5)
```

Comment:
should these be hard coded?

Reply:
Was just using this for a testing file for now